### ISSUE TITLE
Replace all 'NA' with 0 in complete DT

### ISSUE THREAD

**Issue by Peter-Pasta** [h2oai/datatable#2663]
Hi I am wondering how I can replace all the 'NA' after joining two DT's. I found a couple of issues here and looked into the docs, but I wasn't able to leverage this into a solution.

The code below replaces all 1 with 0
`DT.replace(1, 0)`

How should I adapt it to replace 'NA'? Or is there maybe an option to change the padding while joining from 'NA' to '0'?
Thank you.

**Comment by samukweku** [comment]
Hi @Peter-Pasta could you share some data. Please share data not pics. It would be helpful to the wider community if you shared this on Stack Overflow instead; github is more for bugs and feature requests

**Comment by Peter-Pasta** [comment]
Hi @samukweku, I asked this question on Stack Overflow with the sample data (taken from here:https://datatable.readthedocs.io/en/latest/manual/comparison_with_rdatatable.html).

This is the link to the question: https://stackoverflow.com/questions/64086620/replace-all-na-with-0-in-complete-dt-python-datatable.

Should I close this issue here?

**Comment by samukweku** [comment]
Hi @Peter-Pasta , I have answered the question on Stack Overflow. I think it is ok to close the issue here.

**Comment by Peter-Pasta** [comment]
Hi @samukweku, thank you very much for answering this question. As you probably have seen I marked your answer on Stack Overflow as the solution. One interesting thing I noticed is that `replace()` seems to be sensitive regarding the column type (in my case whether the column values are int or float.) 

1. replace NA in int columns
`DT.replace(None, 0 )` 
2. replace NA in float columns
`DT.replace(None, 0.0)`

Maybe it would be a helpful functionality if datatable automatically detects that a 'int 0' is supposed to be used as a replacement in a float column and converts it to '0.0'.
